hw4/04-templates.js

```diff
@@ -1,62 +1,171 @@
-const express = require('express');
```

Comment:
Looks great to me! 
- `/capitals` is formatted correctly and listed alphabetically.
- `/populous` is also formatted correctly for US with commas and listed by decreasing population.
- `/regions` is also also formatted correctly and listed in descending order.